.circleci/config.yml

```
@@ -1175,6 +1184,20 @@ jobs:
     steps:
       - soltest_all
 
+  t_ubu_soltest_deprecated_evm_versions: &t_ubu_soltest_deprecated_evm_versions
+    <<: *base_ubuntu2204_large
+    parallelism: 50
+    steps:
+      - checkout
+      - attach_workspace:
+          at: build
+      - run_soltest_all:
+          evm_versions: "homestead byzantium"
```

You don't really need quotes here.

```suggestion
          evm_versions: homestead byzantium
```